.github/composite-actions/build-modified-postgres/action.yml

```diff
@@ -3,7 +3,7 @@ inputs:
   engine_branch:
     description: 'Engine Branch'
     required: no
-    default: 'latest'
+    default: 'jira-babel-3801'
```

Comment:
We don't need these temp changes as branches name matches b/w engine and extension.